docs/PRETTIER.md

```diff
@@ -0,0 +1,54 @@
+# Prettier
+
+## Adding Prettier to an App
+
+Prettier is available in @trunkclub/build version 7.2 and above. It will require a
+couple dependencies and a small update to the app's `package.json`.
+
+### Before Proceeding
+
+It might beneficial to run Prettier once on the app ahead of time,
+and then committing the formatted changes.
+
+This will ensure all diffs going forward are cleaner and easier to understand.
+Otherwise, diffs will have a lot of noise the first time a changed file is reformatted.
+
+This can be by running the following from your project root:
+
+```bash
+node_modules/.bin/tcweb-build prettier "src/**/*.{js,es6}"
```

Comment:
If someone is using `yarn` (which I think everyone is now) they should be able to issue 

```bash
yarn tcweb-build prettier "src/**/*.{js,es6}
```